### DISCUSSION TITLE
Where is the next.js offical website source code?

### DISCUSSION THREAD

**Discussion by jeffreys-cat** [vercel/next.js#64542] [Help] [answered]
### Summary

The next.js repo only has docs

**Comment by Icegreeen** [comment, marked as answer]
> ### Summary
> The next.js repo only has docs
> 
> ### Additional information
> _No response_
> 
> ### Example
> _No response_

Not in the organization. I believe it is not open source, for content protection reasons.

**Reply by jeffreys-cat** [reply]
Thanks!
